js/src/scatter.ts

```diff
@@ -319,11 +415,12 @@ class ScatterView extends widgets.WidgetView {
         }
         const sprite = geo.endsWith("2d");
         const buffer_geo = new THREE.BufferGeometry().fromGeometry(this.geos[geo]);
+        buffer_geo.computeVertexNormals();
```

Comment:
Do we always need this? Does this impact performance? Can we instead move this to the constructor?

Reply:
Yes, I reckon this line can be moved to the constructor.